.github/workflows/rdev-update-for-pr.yml

```diff
@@ -56,6 +56,19 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 1
+      - name: Get previous image digests for each repo
+        id: "get_digests"
+        run: |
+          ECR_REPOS=(corpora-frontend corpora-backend corpora-upload-failures corpora-upload-success \
+            dataset-submissions corpora-upload wmg-processing cellguide-pipeline)
+          IMAGE_DIGESTS=()
+          for ecr_repo in "${ECR_REPOS[@]}"; do
+            IMAGE_DIGESTS+=(`aws ecr list-images --repository-name $ecr_repo --filter tagStatus=TAGGED \
+              | jq -r --arg STACK_NAME ${STACK_NAME} '[.imageIds[] \
+              | select(.imageTag==$STACK_NAME)][0].imageDigest'`)
+          done
+          echo "ECR_REPOS=${ECR_REPOS[@]}" >> $GITHUB_OUTPUT
+          echo "IMAGE_DIGESTS=${IMAGE_DIGESTS[@]}" >> $GITHUB_OUTPUT
```

Comment:
Does this code run each repo on each matrix slice? Yes. But that doesn't matter. It's quick, and whenever an image digest is missing, it just ignores that. Deletion attempts of images that don't exist aren't a problem.

Reply:
Can update to be a bit more streamlined later, but that will require separating out into a different job and configuring outputs. Might consider making it its own workflow later. Minor tech debt for another day.

Reply:
Nvm, went ahead and fixed this

Reply:
Nvm, went ahead and fixed this